src/main/java/de/tum/in/www1/artemis/config/ArtemisSpringManagedContext.java

```diff
@@ -20,7 +20,7 @@ public ArtemisSpringManagedContext(ApplicationContext applicationContext) {
     @Override
     public Object initialize(Object obj) {
         String type = obj != null ? obj.getClass().getName() : "null";
-        log.debug("Initialize obj {} of type {}", obj, type);
+        log.debug("Initialize object of type {}", type);
```

Comment:
can we use the same semantics as in the line above? so that objects that are not null can still be logged?

Reply:
The problem is not logging a null-object, but logging an Object that throws a NullPointerException when executing `toString()`. But I need to check if your change regarding testing fixed that, since I also noticed the issue while running server tests.

Reply:
Yes, this is an issue in the implementation of `com.hazelcast.map.impl.operation.ClearOperation`.
I could also just check for `instanceof ClearOperation` and ignore this specific class here.